server/.env.example

```diff
@@ -1,5 +1,5 @@
 JWT_SECRET='dummy secret'
 SENDGRID_API_KEY='replace-with-api-key-from-sendgrid'
-AUTH0_DOMAIN = 'get the domain from auth0.con'
-AUTH0_API_EXPLORER_CLIENTID = 'get the id from auth0.con'
-AUTH0_API_EXPLORER_SECRET= 'get the id from auth0.com' 
\ No newline at end of file
+AUTH0_DOMAIN = 'https://gardenz.au.auth0.com/'
+AUTH0_API_EXPLORER_CLIENTID = '2cxnaycUS9BKgNlesVGlfIGIQZjVxPTu'
+AUTH0_API_EXPLORER_SECRET= 'WSWxcWFQNZWgUFn_ojRwYuraFkynxs6yFwu-SHMnl1AvzXt4JE-cfN2y-B5Aw_Pa'
```

Comment:
These changes should only be made to your local `.env` file ... NOT `.env.example`. This file will be committed and it now contains secrets 😬 Just undo the changes to this file.

Reply:
Oops!